asteroid/engine/solver.py

```diff
@@ -52,37 +66,38 @@ def __init__(self, data, model, loss_class, optimizer, training_conf):
 
     def _reset(self):
         """ Extract saved model from `continue_from` path. """
-        if self.continue_from:
-            print('Loading checkpoint model {}'.format(self.continue_from))
-            pack = torch.load(self.continue_from)
+        if self.tr_conf['continue_from']:
+            print('Loading checkpoint model '
+                  '{}'.format(self.tr_conf['continue_from']))
+            pack = torch.load(self.tr_conf['continue_from'])
             self.model.module.load_state_dict(pack['state_dict'])
             self.optimizer.load_state_dict(pack['optim_dict'])
-            self.start_epoch = int(pack.get('epoch', 1))
+            self.start_epoch = int(pack['infos'].get('epoch', 1))
             self.tr_loss[:self.start_epoch] = pack['tr_loss'][:self.start_epoch]
             self.cv_loss[:self.start_epoch] = pack['cv_loss'][:self.start_epoch]
         else:
             self.start_epoch = 0
         os.makedirs(self.save_folder, exist_ok=True)
 
     def train(self):
-        for epoch in range(self.start_epoch, self.epochs):
+        for epoch in range(self.start_epoch, self.tr_conf['epochs']):
             print("Training...")
             self.model.train()
             start = time.time()
-            tr_avg_loss = self._run_one_epoch(epoch)
+            tr_avg_loss = self.run_one_epoch(epoch)
             print('-' * 85)
             print('Train Summary | End of Epoch {0} | Time {1:.2f}s | '
                   'Train Loss {2:.3f}'.format(epoch + 1, time.time() - start,
                                               tr_avg_loss))
             print('-' * 85)
-            if self.checkpoint:
+            if self.tr_conf['checkpoint']:
```

Comment:
Should we use get function here? Just to be consistent

Reply:
`get` is called when the key might not be here. In this case, the key is here.